onnxruntime/core/graph/model.cc

```diff
@@ -605,13 +624,19 @@ common::Status Model::LoadFromOrtFormat(const fbs::Model& fbs_model,
   LOAD_STR_FROM_ORT_FORMAT(model->model_proto_, producer_version, fbs_model.producer_version());
   LOAD_STR_FROM_ORT_FORMAT(model->model_proto_, domain, fbs_model.domain());
   LOAD_STR_FROM_ORT_FORMAT(model->model_proto_, doc_string, fbs_model.doc_string());
+  if (fbs_model.graph() && fbs_model.graph()->doc_string()) {
+    model->model_proto_.mutable_graph()->set_doc_string(fbs_model.graph()->doc_string()->c_str());
+  }
   model->model_proto_.set_model_version(fbs_model.model_version());
   model->model_proto_.set_ir_version(fbs_model.ir_version());
 #else
   experimental::utils::LoadStringFromOrtFormat(model->producer_name_, fbs_model.producer_name());
   experimental::utils::LoadStringFromOrtFormat(model->producer_version_, fbs_model.producer_version());
   experimental::utils::LoadStringFromOrtFormat(model->domain_, fbs_model.domain());
   experimental::utils::LoadStringFromOrtFormat(model->doc_string_, fbs_model.doc_string());
+  if (fbs_model.graph() && fbs_model.graph()->doc_string()) {
+    model->graph_doc_string_ = fbs_model.graph()->doc_string()->c_str();
+  }
   model->model_version_ = fbs_model.model_version();
```

Comment:
Can we just save the graph doc string in the model metadata instead of making changes to the Graph class as well? We only want the top level docstring I assume, so it's not relevant to a subgraph.

Reply:
You mean - don't involve the flatbuffers Graph message and keep it all in model even if it is "semantically" associated with Graph ?

Reply:
Pushed an iteration

Reply:
Yeah. It's semantically associated with the top level Graph only (and not any subgraphs), so it could go in either. Keeping it in the model metadata is cleaner though.